pandas/compat/numpy/function.py

```
@@ -288,20 +288,7 @@ def validate_take_with_convert(convert, args, kwargs):
 
 
 def validate_transpose_for_generic(inst, kwargs):
-    try:
```

This previously was only modifying the exception message for Panel objects before raising again, so I simplified to just remove the try...except altogether and let it raise as is

---

If it’s a one liner now, Is this still needed as a separate function?

---

Yea could probably just call this directly now; I'll give it a look tonight and repush